### PR TITLE
HWStream

### PR DESCRIPTION
**Pull request by qpuhuk**
HW is done

### CODE REVIEW

File: src/main/java/AlexLink/HomeWork/HomeworkStream/Task2/Task2.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import java.util.Arrays;
+
+public class Task2 {
+    static final String TEXT = "Задать массив строк. Используя средства Stream API отсортировать строки в\n" +
+            "лексикографическом порядке по возрастанию и убыванию";
+
+    public static void main(String[] args) {
+        String[] list = TEXT.split("[.,]?\\s+");
+        Arrays.stream(list)
+                .sorted(String::compareToIgnoreCase)
```
Comment (atlamax) [inline src/main/java/AlexLink/HomeWork/HomeworkStream/Task2/Task2.java:15]: Try this:
.sorted(Comparator.naturalOrder())

File: src/main/java/AlexLink/HomeWork/HomeworkStream/Task2/Task2.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+                .forEach(System.out::println);
+        System.out.println("******************************************");
+        Arrays.stream(list)
+                .sorted((a, b) -> b.compareToIgnoreCase(a))
```
Comment (atlamax) [inline src/main/java/AlexLink/HomeWork/HomeworkStream/Task2/Task2.java:19]: Try this:
.sorted(Comparator.reverseOrder())

**Comment by atlamax** [comment]
@qpuhuk 
Approved. Merge by yourself after make changes in Task2.

File: src/main/java/AlexLink/HomeWork/HomeworkStream/Task2/Task2.java (hunk 1 above)
Comment (qpuhuk) [reply src/main/java/AlexLink/HomeWork/HomeworkStream/Task2/Task2.java:15]: fixed

File: src/main/java/AlexLink/HomeWork/HomeworkStream/Task2/Task2.java (hunk 2 above)
Comment (qpuhuk) [reply src/main/java/AlexLink/HomeWork/HomeworkStream/Task2/Task2.java:19]: Fixed, I just wanted to do it with ignore case

**Pull request merged**